db/change_cache.go

```diff
@@ -309,7 +309,7 @@ func (c *changeCache) waitForSequenceWithMissing(sequence uint64) {
 
 // Given a newly changed document (received from the tap feed), adds change entries to channels.
 // The JSON must be the raw document from the bucket, with the metadata and all.
-func (c *changeCache) DocChanged(docID string, docJSON []byte, seq uint64, vbNo uint16) {
+func (c *changeCache) DocChanged(docID string, docJSON []byte, seq uint64, vbNo uint16, dataType uint8) {
```

Comment:
Each bit flag in the collection is already a constant (https://github.com/couchbase/sync_gateway/pull/2481/files#diff-e98023f9feb52d7032a97e0afc9baf7cR25) - is that what you're asking?